impl/src/bitfield/expand.rs

```diff
@@ -431,7 +431,7 @@ impl BitfieldStruct {
                     pub fn from_bytes(
                         bytes: [::core::primitive::u8; #next_divisible_by_8 / 8usize]
                     ) -> ::core::result::Result<Self, ::modular_bitfield::error::OutOfBounds> {
-                        if bytes[(#next_divisible_by_8 / 8usize) - 1] >= (0x01 << (8 - (#next_divisible_by_8 - #size))) {
+                        if bytes[(#next_divisible_by_8 / 8usize) - 1] as u16 >= (0x01 << (8 - (#next_divisible_by_8 - #size))) {
```

Comment:
please use `::core::primitive::u16` instead for better macro hygiene

Reply:
Done